src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -3,25 +3,31 @@
 /* eslint-disable import/no-unresolved */
 
 import * as React from 'react';
+<<<<<<< Updated upstream
 import {FlatList, StatusBar, View} from 'react-native';
+=======
+import {AppState, FlatList, View} from 'react-native';
+>>>>>>> Stashed changes
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a merge conflict that hasn't been resolved. Please resolve this before proceeding.
```diff
- <<<<<<< Updated upstream
- import {FlatList, StatusBar, View} from 'react-native';
- =======
- import {AppState, FlatList, View} from 'react-native';
- >>>>>>> Stashed changes
+ import {AppState, FlatList, StatusBar, View} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->